src/workerd/api/crypto/asymmetric.c++

```diff
@@ -1065,8 +1055,8 @@ kj::Own<EVP_PKEY> rsaJwkReader(SubtleCrypto::JsonWebKey&& keyDataJwk) {
   // RSA_set0_*() transfers BIGNUM ownership to the RSA key, so we don't need to worry about
   // calling BN_free().
   OSSLCALL(RSA_set0_key(rsaKey.get(),
-      BN_bin2bn(modulus.begin(), modulus.size(), nullptr),
-      BN_bin2bn(publicExponent.begin(), publicExponent.size(), nullptr),
+      toBignumUnowned(modulus),
```

Comment:
I wonder if we should add `T* kj::Own<T>::disown()` rather than create separate functions for such use-cases?

Reply:
Yeah, I was considering the same thing. The ability to release ownership is the only reason why I didn't eliminate the `std::unique_ptr` use here entirely.